addons/l10n_ke_edi_tremol/models/product.py

```diff
@@ -30,3 +30,13 @@ class ProductProduct(models.Model):
         help="Product code description needed in case of not 16%. ",
         readonly=False,
     )
+    fiscal_country_codes = fields.Char(
+        string='Fiscal Country Codes',
+        compute='_compute_fiscal_country_codes',
+    )
+
+    @api.depends('company_id')
+    @api.depends_context('allowed_company_ids')
+    def _compute_fiscal_country_codes(self):
+        for product in self:
+            product.fiscal_country_codes = product.product_tmpl_id.fiscal_country_codes
```

Comment:
Doesn't look like you use the dependences?
Isn't the field accessible already since it is a `_inherits`?